demo.yml

```diff
@@ -10,9 +10,12 @@
 
   vars:
     # These will need changing based on your project
-    demo_key: jenkins
-    demo_sg: demo-sg-jenkins
+    demo_key: ansible-vm-key
+    demo_sg: demo-sg-ansible
     demo_security_groups: default,{{ demo_sg }} # don't add spaces here!
+    net_id: c55bc796-841f-4704-a1a2-8f29bb9a699a
+    net_name: project_2001316
+    path_to_key: "/home/jtahir/.ssh/id_rsa.pub" # path to local pc public key 
```

Comment:
Perhaps, leave this empty so it fails if the user does not fill it up. It is a bit more user friendly to use:

 ansible-playbook demo.yml -e path_to_key=.....pub